source/fundamentals/data-formats/serialization.txt

```diff
@@ -0,0 +1,190 @@
+.. _fundamentals-kotlin-serialization:
+
+====================
+Kotlin Serialization
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+The Kotlin driver supports the ``kotlinx.serialization`` library for serializing 
+and deserializing Kotlin objects. 
+
+The driver provides an efficient ``Bson`` serializer that can can be used with 
+``@Serializable`` classes to handle the serialization of Kotlin objects to BSON data. 
+The ``bson-kotlinx`` library also supports :ref:`custom codecs <kotlin-custom-codec>`
+with configurations such as encoding defaults, encoding nulls, and defining class 
+discriminators.
+
+.. note::
+
+    To use the ``Codec`` interface to specify custom encoding and decoding of 
+    Kotlin objects to BSON data, see the :ref:`Codecs <fundamentals-codecs>` 
+    documentation.
```

Comment:
Two things:
1. I think the text here is one space too indented
2. Same as my comment on the codecs page, why would users choose one approach over the other?